.github/workflows/ci.yml

```
@@ -29,4 +29,7 @@ jobs:
       run: mage build
 
     - name: Test
-      run: mage test
+      run: mage CITest
+
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
```

```suggestion
      uses: codecov/codecov-action@v3
```

as per https://github.com/codecov/example-go/blob/main/.github/workflows/ci.yml